crates/subspace-core-primitives/src/lib.rs

```diff
@@ -531,3 +532,63 @@ pub fn bidirectional_distance<T: num_traits::WrappingSub + Ord>(a: &T, b: &T) ->
     // Find smaller diff between 2 directions.
     diff.min(diff2)
 }
+
+pub use construct_uint::U256;
```

Comment:
Hmm, maybe merge this line to the top `use` section as the convention in our codebase is to put `use` in one single place?